src/index.ts

```diff
@@ -65,24 +75,20 @@ v1.post('/command', verifyTokenMiddleware, (req, res) => {
 
 function verifyTokenMiddleware(req, res, next: NextFunction) {
 	const token = req.body.token;
-	// if (!token)
-	// 	return res.status(403).send('Could not find token in request body');
-
-	// jwt.verify(token, JWT_SECRET, (err, decoded) => {
-	// 	if (err) {
-	// 		if (err.name === 'TokenExpiredError')
-	// 			return res.status(403).send(`Token expired at ${err.expiredAt}`);
-
-	// 		return res.status(403).send('Invalid Credentials');
-	// 	}
-
-	// 	req.body.hashedServerId = decoded.hashedServerId;
-	// 	next();
-	// });
 
 	const decoded = verifyToken(token);
 	if (decoded === 'NoToken')
```

Comment:
This should probably just check if status does not indicate success and if it doesnt switch on the response (decoded) instead of multiple ifs

Reply:
Yeah, I thought that too after I changed it and forgot before I pushed